src/Chart.php

```diff
@@ -10,46 +10,49 @@
 use Atk4\Ui\View;
 
 /**
- * ChartJS 2.7.x documentation https://www.chartjs.org/docs/2.7.3/
- * ChartJS 3.9.1 documentation https://www.chartjs.org/docs/3.9.1/.
+ * ChartJS 3.9.1 documentation https://www.chartjs.org/docs/3.9.1/
+ * Chart examples https://www.chartjs.org/docs/latest/samples/information.html.
  */
 class Chart extends View
 {
     /** @var string HTML element type */
     public $element = 'canvas';
 
-    /** @var string Type of chart - bar|pie etc. */
-    public $type;
+    /** @var string Type of chart - bar|pie etc. See ChartType class */
```

Comment:
```suggestion
    /** Type of chart - bar|pie etc. See ChartType class */
```

can/should `Chart` be abstract?

`ChartType` constants can be defined in `Chart`

Reply:
I didn't make Chart class abstract because it can be used as is too to create some custom charts in future.
Kind of like this `new Chart(['type' => ChartType::TYPE_BAR])` or `new Chart(['type' => 'custom_type'])`.